config/settings/development.yml

```diff
@@ -655,7 +655,7 @@ kafka_producer:
   aws_role_arn: "arn:aws:iam::123456789012:role/role-name" # this is an example
   broker_urls: ["localhost:19092"] # for local Kafka cluster in Docker
   sasl_mechanisms: 'GSSAPI'
-  schema_registry_url: "http://localhost:8081" # for local Schema Registry in Docker
+  schema_registry_url: "localhost:8081" # for local Schema Registry in Docker
```

Comment:
Hmm, how would this work locally? `https:localhost:8081` would break. Perhaps this is a reason to add the protocol to the settings variable after all ... 🤔